var/spack/repos/builtin/packages/rocalution/package.py

```diff
@@ -146,6 +149,7 @@ class Rocalution(CMakePackage):
             )
 
     depends_on("googletest@1.10.0:", type="test")
+    patch("0003-fix-compilation-for-rocalution-5.2.0.patch", when="@5.2.0:")
```

Comment:
Same comments on patches as above

Reply:
@alalazo  This is already upstreamed and will be part of 5.2.1 release. This specific change was late find when doing the spack build .